### PR TITLE
HPC: Move libsolv test to hpc directory

### PR DESCRIPTION
**Pull request by schlad**
For now I see no reason why to try to make it generic. This should be first well used for the HPC testing

- Related ticket: https://progress.opensuse.org/issues/39080

### CODE REVIEW

**Comment by schlad** [comment]
@asmorodskyi would you mind taking a look please?

File: tests/hpc/changelog [hunk 1]
```diff
@@ -5,4 +5,5 @@ All notable changes to this project will be documented in this file
 ### Added
 Add the change log along with the README
 ### Changed
+Move libsolv test to the HPC group as this should be well crafted for the HPC use
```
Comment (okurz) [inline tests/hpc/changelog]: I think the idea of the changelog is that you would create a new "version" and mention your changes there

**Comment by foursixnine** [comment]
@schlad please try to use more descriptive titles for the Pull requests, makes it easier to understand what it is about from a bird's eye.

**Comment by asmorodskyi** [comment]
> @schlad please try to use more descriptive titles for the Pull requests, makes it easier to understand what it is about from a bird's eye.

For example : "HPC: Move libsolv test to hpc directory"

File: tests/hpc/changelog (hunk 1 above)
Comment (asmorodskyi) [reply tests/hpc/changelog]: depends , I think in this certain case , it is good enough without dive into versions for HPC tests . 
@schlad :+1:  I like the idea

**Comment by schlad** [comment]
> @schlad please try to use more descriptive titles for the Pull requests, makes it easier to understand what it is about from a bird's eye.
sure, sure. I am old. I care what is in the git commit message and not so much in what is on this fancy page. But sure, dully noted. Will comply :-)

File: tests/hpc/changelog (hunk 1 above)
Comment (schlad) [reply tests/hpc/changelog]: > depends , I think in this certain case , it is good enough without dive into versions for HPC tests .

I fully agree. +2

**Comment by okurz** [comment]
@schlad as you are about as old as I am: I am never creating a pull request from the webpage and I always create the PR title automatically from the command line using
https://github.com/okurz/scripts/blob/master/git-pr-last or https://github.com/okurz/scripts/blob/master/git-pr-more which both use
https://github.com/github/hub
So don't waste your time on fancy web pages ;)

**Pull request merged**